extended/jams2015.tex

```diff
@@ -436,7 +438,7 @@ \subsection{Data frames}\label{sec:imp:dataframe}
 JAMS data frame objects interpret all \texttt{time} and \texttt{duration} fields as
 \texttt{timedelta} types.  In addition to facilitating semantic validation ---
 \texttt{time} and \texttt{duration} fields are enforced to contain non-negative values --- 
-this enables pandas to efficiently align and resample multiple annotations with 
+this enables Pandas to efficiently align and resample multiple annotations with 
```

Comment:
`pandas` (the package name) is not capitalized.  See [here](http://pandas.pydata.org/)